app/helpers/value_helper.rb

```diff
@@ -69,9 +69,14 @@ def links_to_collections(args)
     end.join('<br>').html_safe
   end
 
+  # Discussed removing this method because the field should not be modified.
   def value_for_originInfo_date_created_tesim(args)
-    val = Time.parse(args[:document][args[:field]].first)
+    field_value = args[:document][args[:field]].first
+    # Try to parse the string as a time value
+    val = Time.parse(field_value)
```

Comment:
Is it? I think the behavior rubocop is trying to discourage here is using `Time.parse` with a poorly-specified timezone reference, to catch differences like:

```ruby
# Using the system time zone (PT)
irb(main):003:0> Time.parse('2015-01-01')
=> 2015-01-01 00:00:00 -0800

# Using the application time zone (UTC)
irb(main):004:0> Time.zone.parse('2015-01-01')
=> Thu, 01 Jan 2015 00:00:00 UTC +00:00
```

See also this pretty nice guide to time handling in rails:  http://danilenko.org/2012/7/6/rails_timezones/